atd-vzv/src/views/map/Map.js

```diff
@@ -58,6 +59,7 @@ const Map = () => {
   const [cityCouncilOverlay, setCityCouncilOverlay] = useState(null);
   const [isMapDataLoading, setIsMapDataLoading] = useState(false);
   const [crashCounts, setCrashCounts] = useState(null);
+  const [, setPointData] = useState(null);
```

Comment:
This is interesting, we create a state but we don't use it. To me this seems like a function that just triggers a change in the state to re-render via useEffect?

Reply:
Yeah so the map wouldn't rerender unless I updated a state value. It kind of feels hacky, but if I don't try and update a state value, the animated icon won't have the 'animated' effect. If you look at where I'm calling `setPointData`, I'm passing in an empty object. From what I read somewhere, because it's an object, Mapbox interprets it as a new value and the whole map gets rerendered causing the animated icon to animate. If you change the empty object to a value such as a number or string, the map doesn't rerender. I'm not sure how correct what I'm saying is, but that's what I found on google.

Reply:
@jgabitto It's not hacky it's genious! 👍 💯  Who knows if this is best practice but at least it is _working_ code. 😄  I love that code, but if you don't mind to add some commentary as to why that code was written that way so anyone who has to work on it will know what it's doing and why.